changelog/unreleased/changes/2033.md

```diff
@@ -0,0 +1,2 @@
+The Arrow dependency has been increased to 6.0, which is the most recent stable
+version.
```

Comment:
Please use active voice and present tense for changelog entries, e.g., "Building VAST now requires Arrow >= 6.0."